OpenRA.Mods.Common/Traits/Render/WithSpriteBody.cs

```diff
@@ -70,7 +70,7 @@ protected WithSpriteBody(ActorInitializer init, WithSpriteBodyInfo info, Func<in
 
 			if (info.StartSequence != null)
 				PlayCustomAnimation(init.Self, info.StartSequence,
-					() => PlayCustomAnimationRepeating(init.Self, info.Sequence));
+					() => DefaultAnimation.PlayRepeating(NormalizeSequence(init.Self, info.Sequence)));
```

Comment:
What is wrong with not changing this? (It should work already after you changed `PlayCustomAnimationRepeating`. Not that this is a blocker, just curious.)

Reply:
Mostly for consistency with the 'else' below, and semantics (the default sequence is not a custom animation, afterall).